### PR TITLE
update how to create docker secrets for builds

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -1589,23 +1589,25 @@ able to launch a build whenever a `*BuildConfig*` is updated.
...
 
-Supply the *_.dockercfg_* file with valid Docker Registry credentials in order to
+Supply the *_.docker/config.json_* file with valid Docker Registry credentials in order to
 push the output image into a private Docker Registry or pull the builder image
 from the private Docker Registry that requires authentication. For the OpenShift
 Docker Registry, you don't have to do this because `*secrets*` are generated
 automatically for you by OpenShift.
 
-The *_.dockercfg_* JSON file is found in your home directory by default and has
+The *_.docker/config.json_* JSON file is found in your home directory by default and has
```
Comment (tpoitras) [inline dev_guide/builds.adoc]: Can probably remove "JSON" now that you've included the file extension in the file name.

**Comment by tpoitras** [comment]
One small nit, other than that LGTM. :+1:

**Comment by bparees** [comment]
@tpoitras updated, thanks.

**Pull request merged**